src/components/Footer/index.js

```diff
@@ -8,35 +8,35 @@ const Footer = () => {
       <div className={classes.allInfoContainer}>
         <div className={classes.infoContainer}>
           <p className={classes.infoHeader}>About</p>
-          <a className={classes.infoLink}>Overview</a>
-          <a className={classes.infoLink}>Searching the Index</a>
-          <a className={classes.infoLink}>Adding Projects to the Index</a>
-          <a className={classes.infoLink}>Index Contributors</a>
+          <a href='#' className={classes.infoLink}>Overview</a>
+          <a href='#' className={classes.infoLink}>Searching the Index</a>
+          <a href='#' className={classes.infoLink}>Adding Projects to the Index</a>
+          <a href='#' className={classes.infoLink}>Index Contributors</a>
         </div>
         <div className={classes.infoContainer}>
           <p className={classes.infoHeader}>Tag Generator</p>
         </div>
         <div className={classes.infoContainer}>
           <p className={classes.infoHeader}>Communicate with Us</p>
-          <a className={classes.infoLink}>FAQ</a>
+          <a href='#' className={classes.infoLink}>FAQ</a>
         </div>
         <div className={classes.infoContainer}>
           <p className={classes.infoHeader}>Support Us</p>
-          <a className={classes.infoLink}>Donate</a>
-          <a className={classes.infoLink}>Share the CTI</a>
-          <a className={classes.infoLink}>Volunteer with Us</a>
+          <a href='#' className={classes.infoLink}>Donate</a>
+          <a href='#' className={classes.infoLink}>Share the CTI</a>
+          <a href='#' className={classes.infoLink}>Volunteer with Us</a>
         </div>
         <div className={classes.infoContainer}>
           <p className={classes.infoHeader}>Follow Us</p>
-          <img className={classes.margin} src='/images/fb.svg' alt='instagram logo'/>
-          <img className={classes.margin} src='/images/fb.svg' alt='twitter logo'/>
-          <img className={classes.margin} src='/images/fb.svg' alt='facebook logo'/>
-          <img src='/images/fb.svg' alt='github logo'/>
+          <a href='#'><img className={classes.margin} src='/images/insta-logo.svg' alt='instagram logo'/></a>
+          <a href='#'><img className={`${classes.margin} ${classes.twitter}`} src='/images/twitter-logo.svg' alt='twitter logo'/></a>
+          <a href='#'><img className={classes.margin} src='/images/fb-logo.svg' alt='facebook logo'/></a>
+          <a href='https://github.com/civictechindex'><img src='/images/github-logo.svg' alt='github logo'/></a>
```

Comment:
Can we make a global JSON file and keep all the links there and import them wherever required?
This will make sure that any changes/additions in the future will have to touch just the JSON file.

Reply:
For sure, I'll add that in. Thank you